.pre-commit-config.yaml

```diff
@@ -1,15 +1,41 @@
 repos:
-  - repo: https://github.com/golangci/golangci-lint
-    rev: v1.59.1
-    hooks:
-      - id: golangci-lint-full
   - repo: https://github.com/tekwizely/pre-commit-golang
     rev: v1.0.0-rc.1
     hooks:
       - id: go-mod-tidy-repo
-      - id: go-fmt
-      - id: go-test-repo-mod
   - repo: https://github.com/gruntwork-io/pre-commit
     rev: v0.1.23
     hooks:
-      - id: helmlint
\ No newline at end of file
+      - id: helmlint
+  - repo: local
+    hooks:
+      - id: fmt
+        name: fmt
+        entry: make fmt
+        language: system
+        types: [ file ]
+        pass_filenames: false
+  - repo: local
+    hooks:
+      - id: lint
+        name: golangci-lint
+        entry: make lint
```

Comment:
previously, golangci-lint version was the same across all environments, now we open possibility for drift.